docs/examples.rst

```diff
@@ -306,3 +306,49 @@ Output:
+
+    # Address field should be the checksum address at which the ERC20 contract was deployed
+    erc20 = w3.eth.contract(address='0x89205A3A3b2A69De6Dbf7f01ED13B2108B2c43e7', abi=ERC20_ABI)
+
+Fetch various data about the current state of the ERC20 contract.
+
+.. code-block:: python
+
+    name = erc20.functions.symbol().name()
+    # -> 🦄
+    symbol = erc20.functions.symbol().call()
+    # -> 🦄
+    decimals = erc20.functions.decimals().call()
+    # -> 1
+    total_supply = erc20.functions.totalSupply().call()
+    # -> 2038 (at time of writing)
+
+Get the balance of an account address. (token balance is *always* an integer)
+
+.. code-block:: python
+
+    balance = erc20.functions.balanceOf('0xD1220A0cf47c7B9Be7A2E6BA89F429762e7b9aDb').call()
+    # -> 1
+
+Calculate the token count from the token balance of an account address. (token count *may* be a fraction)
+
+.. code-block:: python
+    
+    count = balance / (10 ** decimals)
```

Comment:
Try `Decimal()` instead of `float` here, with a high enough precision to avoid any rounding error. Most applications expect exact results, even if fractional.